tests/NLog.UnitTests/LogFactoryTests.cs

```diff
@@ -96,6 +94,32 @@ public void InvalidXMLConfiguration_ThrowErrorWhen_ThrowExceptionFlagIsSet()
             Assert.True(ExceptionThrown);
         }
 
+        [Fact]
+        public void InaccessibleNLogConfigFileDoesNotThrow()
+        {
+            var tempDirectory = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("N"));
+            var configFile = Path.Combine(tempDirectory, "NLog.config");
+
+            try
+            {
+                Directory.CreateDirectory(tempDirectory);
+                File.WriteAllText(configFile, "<nlog />");
+                LogFactory logFactory = new LogFactory();
+                logFactory.SetCandidateConfigFilePaths(new[] { configFile });
+                using (var fileLock = new FileStream(configFile, FileMode.Open, FileAccess.ReadWrite, FileShare.None))
+                {
+                    var loggingConfig = logFactory.Configuration;
+                    Assert.Null(loggingConfig);
+                    Assert.ThrowsAny<Exception>(() => logFactory.LoadConfiguration(configFile));
```

Comment:
i'm a bit confused. The method title says "DoesNotThrow", but the assert says the contradiction.

Reply:
Maybe we should split this into 2 unit tests, or rename it. Currently, I don't know what to do with this. I kinda brain overload.

Reply:
I think these are 2 tests.  --> https://github.com/snakefoot/NLog/pull/4